src/lib/crypto/kimchi_bindings/wasm/src/plonk_proof.rs

```diff
@@ -852,136 +852,3 @@ pub mod fq {
         Fq
     );
 }
-
-#[cfg(test)]
```

Comment:
looks like these tests are not exercised, since it's outdated. Also probably not a good place for that test?